OutboundCallReminder/OutboundCallReminder/OutboundCallReminder.cs

```diff
@@ -145,16 +145,17 @@ private async Task PlayAudioAsync()
                 // Preparing data for request
                 var playAudioRequest = new PlayAudioOptions()
                 {
+                    AudioFileUri = new Uri(callConfiguration.AudioFileUrl),
                     OperationContext = Guid.NewGuid().ToString(),
                     Loop = true,
                 };
 
                 Logger.LogMessage(Logger.MessageType.INFORMATION, "Performing PlayAudio operation");
-                var response = await callConnection.PlayAudioAsync(new Uri(callConfiguration.AudioFileUrl), playAudioRequest, reportCancellationToken).ConfigureAwait(false);
+                var response = await callConnection.PlayAudioAsync(playAudioRequest, reportCancellationToken).ConfigureAwait(false);
 
                 Logger.LogMessage(Logger.MessageType.INFORMATION, $"PlayAudioAsync response --> {response.GetRawResponse()}, Id: {response.Value.OperationId}, Status: {response.Value.Status}, OperationContext: {response.Value.OperationContext}, ResultInfo: {response.Value.ResultInfo}");
 
-                if (response.Value.Status == CallingOperationStatus.Running)
+                if (response.Value.Status == OperationStatus.Running)
```

Comment:
seems this PR will revert back the latest changes on outbound call. could you double check?

Reply:
we got these OutboundCallReminder changes after fetching the latest changes from **azure-samples:main** branch.
Also these changes are working with latest beta2 and beta3 packages